src/main/java/com/amplitude/api/AmplitudeClient.java

```diff
@@ -334,11 +334,6 @@ public synchronized AmplitudeClient initializeInternal(
             if (!initialized) {
                 // this try block is idempotent, so it's safe to retry initialize if failed
                 try {
-                    if (instanceName.equals(Constants.DEFAULT_INSTANCE)) {
-                        AmplitudeClient.upgradePrefs(context);
```

Comment:
do you want to clean up these methods too? `upgradePrefs` and `upgradeSharedPrefsToDB`

Reply:
Going to clean up everything, including the tests @djih

Reply:
In a different branch and PR